packages/loopring_v3/contracts/amm/LoopringAmmFactory.sol

```diff
@@ -22,28 +22,27 @@ contract LoopringAmmFactory is ReentrancyGuard
         poolImplementation = _poolImplementation;
     }
 
-    function createPool(AmmData.PoolConfig calldata config)
+    function createPool(
+        uint salt,
+        AmmData.PoolConfig calldata config
```

Comment:
LPToken need to be registered in Exchange, so LPTokenId need to be added in PoolConfig.